antora/modules/ROOT/nav.adoc

```diff
@@ -17,6 +17,7 @@
 ** xref:{chapters}portability_initiative.adoc[]
 ** xref:{chapters}vulkan_cts.adoc[]
 ** xref:{chapters}development_tools.adoc[]
+** xref:{chapters}ide.adoc[]
```

Comment:
this should be the order it is in README/Guide (above Vulkan Profiles)

or make the README/Guide.adoc match here

personally I think the spot here make sense and the READEME/Guide should be adjusted